authlib/integrations/base_client/async_app.py

```diff
@@ -73,7 +73,12 @@ async def _on_update_token(self, token, refresh_token=None, access_token=None):
     async def load_server_metadata(self):
         if self._server_metadata_url and '_loaded_at' not in self.server_metadata:
             async with self.client_cls(**self.client_kwargs) as client:
-                resp = await client.request('GET', self._server_metadata_url, withhold_token=True)
+                resp = await client.request(
+                    'GET',
+                    self._server_metadata_url,
+                    withhold_token=True,
+                    **self.client_kwargs,
+                )
```

Comment:
❓ Are there no tests to update for this case?

Reply:
There are no tests for `fetch-jwk` or for `load_server_metadata` for neither the sync app or the async app. I added this commit to make sure the async app works the same as the sync app. I did add regression tests for the sync app though as it's important for us.